### DISCUSSION TITLE
Add IIRC scenario (?)

### DISCUSSION THREAD

**Discussion by vlomonaco** [ContinualAI/avalanche#397] [Feature Request]
We should check how to integrate this scenario in our framework: https://chandar-lab.github.io/IIRC
The difference its mostly in the evaluation part, where a sample have multiple correct labels.

**Comment by AntonioCarta** [comment]
My guess from a quick look is that you only need a different Accuracy metric, the rest should work as is. It could be a good test to check if avalanche works with different metrics and ensure that the accuracy is not hardcoded anywhere (as it was in the old eval plugin).

**Comment by ggraffieti** [comment]
I agree with @AntonioCarta, I also quick-checked the site and seems that we can smoothly integrate the scenario just using an ad hoc accuracy metric. 
If I understood well, seems every data point has 2 labels, general and fine, and you have to guess one of the two given the other one. I don't understand how a strategy can use the given label, but in any case the how-to will be delegated to the particular strategy.
